tests/cases/fourslash/completionListInObjectLiteral11.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference path="fourslash.ts" />

```

Comment:
this looks like a test case from https://github.com/microsoft/TypeScript/issues/44428 but you have not linked that issue in this PR - should it get linked? :p

Reply:
I am not sure
When I run this test without the changes made in this PR, it passes anyway
Maybe it was already solved